doc/content/getting_started/installation.md

```diff
@@ -85,12 +115,25 @@ build the code and run the tests. The number in that flag can be changed to the
 number of physical and virtual cores on the workstation being used to build FENIX.
 !alert-end!
 
+!alert! note title=Speeding up OpenMC/DAGMC/MOAB builds
+If the build of Cardinal third-party libraries OpenMC, DAGMC, and MOAB is too slow on your machine,
+you can attempt to pass the number of `make` cores using the `MAKEFLAGS` variable. This can do a
+better job enabling proper parallelism in nested `make` builds (as is the case with Cardinal
+dependencies). To use this variable, simply perform:
+
+```bash
+make MAKEFLAGS=-j8
+```
+
+instead of the `make` commands highlighted above.
+!alert-end!
+
 If FENIX is working correctly, all active tests will pass. This indicates that
 FENIX is ready to be used and further developed.
 
 ## Update FENIX
 
-FENIX (and its underlying dependencies MOOSE and TMAP8) is under heavy development and is
+FENIX (and its underlying dependencies MOOSE, TMAP8, and Cardinal) is under heavy development and is
 updated on a continuous basis. Therefore, it is important that the local copy of FENIX be periodically
 updated to obtain new capabilities, improvements, and bugfixes. Weekly updates are recommended as,
 at minimum, the MOOSE submodule within FENIX is updated up to several times a week.
```

Comment:
I cannot point to it or make a suggestion there directly, but the instructions to update FENIX should also be updated. 
It currently only update moose, and should update TMAP8 and Cardinal with something  

```
git submodule update moose
git submodule update tmap8
git submodule update cardinal
```
Note that this change should occur in two different places.

Reply:
Addressed in ffa88f9.